light-zk.js/src/account.ts

```diff
@@ -104,7 +104,7 @@ export class Account {
       // burnerSeed can be shared since hash cannot be inverted - only share this for app utxos
       // sharing the burnerSeed saves 32 bytes in onchain data if it is require to share both
       // the encryption and private key of a utxo
-      this.burnerSeed = new BN(bs58.decode(seed)).toBuffer("be", 32);
+      this.burnerSeed = new BN(bs58.decode(seed)).toArrayLike(Buffer, "be", 32);
```

Comment:
we should do another pr where we change this in all places where BN.toBuffer/toArray is used